### ISSUE TITLE
Make the bottom edge of the Toolbar semi-transparent

### ISSUE THREAD

**Issue by vit001** [miguelhincapie/CustomBottomSheetBehavior#37]
![screenshot_2017-02-10-00-35-03](https://cloud.githubusercontent.com/assets/3802799/22819959/86f17034-ef29-11e6-9cc6-6096f866354b.png)

**Comment by miguelhincapie** [comment]
that can be achieve easy using a transparent layout like you did with the bottomSheet (in that one you used a black shadow).

**Comment by vit001** [comment]
Yes, you are absolutely right! I was hoping some other lib user could pick it up, I am also super tied-up with my app right now.
